ui/src/app/lib/atlasmap-data-mapper/utils/mapping-serializer.spec.ts

```diff
@@ -34,7 +34,8 @@ describe('MappingSerializer', () => {
     });
     cfg = ConfigModel.getConfig();
     cfg.clearDocs();
-    cfg.mappings = new MappingDefinition();
+    cfg.mappings = new MappingDefinition(0);
+    cfg.mappingId = 0;
```

Comment:
How does mappingId (config.model) correspond to MappingDefinition.activeMapping??

Reply:
Maybe the name of `mappingId` is confusing, @AbbassFaytaroony @pleacu WDYT about using `mappingDefinitionId` instead? yeah it's long, but less confusing with individual mapping.

Reply:
Yes, we are actually Identifying mapping definition as in front-end terminology. Still afraid however it may cause confusion in the backend where the same entity is called `AtlasMapping`. But yes, `mappingDefinitionId` will be overall more accurate.